docs/Guides/Ecosystem.md

```diff
@@ -181,6 +181,8 @@ section.
   to go!
   A plugin to implement [Lyra](https://github.com/nearform/lyra) search engine
   on Fastify
+- [`@joggr/fastify-prisma`](https://github.com/joggrdocs/fastify-prisma)
```

Comment:
I noticed that the plugin's version is set to `0.0.0`, is that expected?

Reply:
Plugin version is set by release tag: https://www.npmjs.com/package/@joggr/fastify-prisma

Reply:
maybe I'll add a simple commit in CI to update the version in the source package.json.... 😄